pyensembl/gtf_parsing.py

```diff
@@ -121,20 +122,30 @@ def _extend_with_attributes(df):
     extra_columns = {}
     column_order = []
     for i, attr_string in enumerate(df.attribute):
+        # TODO: tokenize attribute strings properly
+        # for now, catch mistaken semicolons by replacing "xyz;" with "xyz"
+        # Required to do this since the GTF for Ensembl 78 has
```

Comment:
It wasn't obvious to me what "mistake semicolons are": can you clarify that these are semicolons that don't actually split key and value? (Therefore clarifying while we're still splitting on ";" below.)